sw/device/silicon_creator/rom/e2e/BUILD

```diff
@@ -182,12 +193,89 @@ opentitan_functest(
         timeout = "eternal",
         rom = "//sw/device/silicon_creator/rom",
     ),
-    deps = [
-        "//sw/device/lib/dif:keymgr",
-        "//sw/device/lib/testing:keymgr_testutils",
-        "//sw/device/lib/testing:otp_ctrl_testutils",
-        "//sw/device/lib/testing/test_framework:ottf_main",
-    ],
+)
+
+rom_e2e_keymgr_init_configs = [
+    {
+        "name": "rom_ext_meas",
+        "value": CONST.TRUE,
+    },
+    {
+        "name": "rom_ext_no_meas",
+        "value": CONST.FALSE,
+    },
+    {
+        "name": "rom_ext_invalid_meas",
+        "value": 0,
+    },
+]
+
+[
+    otp_json(
+        name = "otp_json_keymgr_{}".format(config["name"]),
+        partitions = [
+            otp_partition(
+                name = "OWNER_SW_CFG",
+                items = {
+                    "OWNER_SW_CFG_ROM_KEYMGR_ROM_EXT_MEAS_EN": hex(config["value"]),
+                },
+            ),
+        ],
+    )
+    for config in rom_e2e_keymgr_init_configs
+]
+
+[
+    otp_image(
+        name = "otp_img_keymgr_{}".format(config["name"]),
+        src = "//hw/ip/otp_ctrl/data:otp_json_rma",
+        overlays = STD_OTP_OVERLAYS + [":otp_json_keymgr_{}".format(config["name"])],
+        visibility = ["//visibility:private"],
+    )
+    for config in rom_e2e_keymgr_init_configs
+]
+
+[
+    bitstream_splice(
+        name = "bitstream_keymgr_{}".format(config["name"]),
+        src = "//hw/bitstream:rom",
+        data = ":otp_img_keymgr_{}".format(config["name"]),
+        meminfo = "//hw/bitstream:otp_mmi",
+        tags = ["vivado"],
+        visibility = ["//visibility:private"],
+    )
+    for config in rom_e2e_keymgr_init_configs
+]
+
+[
+    opentitan_functest(
+        name = "rom_e2e_keymgr_init_{}".format(config["name"]),
+        cw310 = cw310_params(
+            bitstream = ":bitstream_keymgr_{}".format(config["name"]),
+            tags = ["vivado"],
+        ),
+        dv = dv_params(
+            rom = "//sw/device/silicon_creator/rom",
+        ),
+        ot_flash_binary = ":rom_e2e_keymgr_init_test",
+        otp = ":otp_img_keymgr_{}".format(config["name"]),
```

Comment:
I don't think this is quite right is it? `otp` param should be fed to `dv_params()` and to `cw310_params()`. Our current implentation of `opentitan_functest`/`dv_params()`/`cw310_params()` sets the OTP param to a default value if not provided. See [here](https://cs.opensource.google/opentitan/opentitan/+/master:rules/opentitan_test.bzl;drc=344398756af305af675e76aa081f6bf16dc8c3bc;l=307) and [here](https://cs.opensource.google/opentitan/opentitan/+/master:rules/opentitan_test.bzl;drc=344398756af305af675e76aa081f6bf16dc8c3bc;l=52).

Reply:
Thanks for knowing/checking, I'll update it!